FV3/wrapper/fv3gfs/wrapper/physics_properties.json

```diff
@@ -463,6 +463,14 @@
         "container": "Sfcprop",
         "dims": ["z_soil", "y", "x"]
     },
+    {
+        "name": "standard_deviation_of_subgrid_orography",
+        "fortran_name": "hprime",
+        "units": "m",
+        "description": "orographic metrics",
+        "container": "Sfcprop",
+        "dims": ["nmtvr", "y", "x"]
```

Comment:
`nmtvr` isn't very descriptive for the dimension, could we name it `orographic_variable`?

Reply:
Feel free to swap orographic for topographic in my comments as you see fit, as long as it's consistent everywhere.

Reply:
My vote would be `orographic` since that's used re: the gravity wave drag parameterization and elsewhere